Lib/_strptime.py

```diff
@@ -565,6 +566,10 @@ def _strptime(data_string, format="%a %b %d %H:%M:%S %Y"):
             hour, minute, second,
             weekday, julian, tz, tzname, gmtoff), fraction, gmtoff_fraction
 
+date_specs = ('%a', '%A', '%b', '%B', '%c', '%d', '%j', '%m', '%U',
+              '%w', '%W', '%x', '%y', '%Y',)
+time_specs = ('%T', '%R', '%H', '%I', '%M', '%S', '%f', '%i', '%s',)
```

Comment:
I don't see any references to `%T`, `%R`, `%i` or `%s` in the docs or elsewhere in the code. What do these represent?